dtfs-central-api/src/types/utilisation-report-status.ts

```diff
@@ -0,0 +1,46 @@
+import { ObjectId } from 'mongodb';
+import { TFMUser } from './users';
+
+export type ReportStatus = 'RECONCILIATION_COMPLETED' | 'REPORT_NOT_RECEIVED';
+
+export type ReportDetails = {
+  month: number;
+  year: number;
+  bankId: string;
+};
+
+type ReportId = {
+  id: string;
+};
+
+type ReportWithStatus = {
+  status: ReportStatus;
+  report: ReportDetails | ReportId;
+};
+
+export type PutReportStatusRequestBody = {
+  user: TFMUser;
+  reportsWithStatus: ReportWithStatus[];
+};
+
+export type ReportFilter =
+  | { _id: ObjectId }
+  | {
+      month: number;
+      year: number;
+      'bank.id': string;
+    };
+
+export type UtilisationReport = {
+  bank: {
+    id: string;
+    name?: string;
```

Comment:
I've added this as optional for the sake of the placeholder report - I'm planning on re-thinking how I implement this though

Reply:
Edit: re slack discussion about DB types, we should have this type represent the DB collection type, so be in `src/types/db-models/utilisation-reports.d.ts`